src/module/module.jl

```diff
@@ -407,6 +408,21 @@ function prune_with_map(I::smodule{spoly{T}}) where T <: Nemo.FieldElem
    return smodule{spoly{T}}(R, r),smatrix{spoly{T}}(R,TT_ptr)
 end
 
+@doc raw"""
+    prune_with_map_projection(id::smodule{spoly{T}}) where T <: Nemo.FieldElem
+
+Returns the module R minimally embedded in a free module such that the
```

Comment:
Should be "Return", not "Returns" (also in `prune_with_map` above). And I guess `R` should be code?
```suggestion
Return the module `R` minimally embedded in a free module such that the
```